### PR TITLE
Updated the Nextcloud dashboard and alerts.

### PR DESCRIPTION
**Pull request by pkudikyala**
**Jira Ticket:** https://new-relic.atlassian.net/browse/NR-183779
**Description:** The existing Nextcloud quickstart relies on a third-party Prometheus exporter, which has become unreliable and is no longer being maintained. To ensure the continued viability of the quickstart, we need to transition to Nri-flex for metric collection.

# Summary

A concise description of the changes being introduced. Please review the pre-merge checklist section to validate this pull request is ready for review and merge. **If it is not ready, please mark the pull request as a draft.**

_The owners of this repo are not experts in the subject matter of the quickstarts. We review for the quickstart to be functional and for security risks. If you are seeking feedback on the content of the quickstart, please seek out a subject matter expert. If you are not an internal NR contributor, we can do our best to connect you with a content reviewer._

## Pre merge checklist

- [ ] Did you check you NRQL syntax? - Does it work?
- [ ] Did you include a Data source and Documentation reference?
- [ ] Are all documentation links publicly accessible?
- [ ] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [ ] Did you check your descriptive content for spelling and grammar errors?
- [ ] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [ ] Does the PR contain a screenshot for each of your dashboards?
- [ ] Do your screenshots show data?
- [ ] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [ ] Did you check that your alerts actually work?

### CODE REVIEW

**Comment by aswanson-nr** [comment]
not stale

**Review by mdumpati — APPROVED** [review]
LGTM

File: alert-policies/nextcloud/CPUUsage.yml [hunk 1]
```diff
@@ -1,11 +1,11 @@
-name: Scrape Errors
+name: CPU Usage
 
 description: |+
-  This alert is triggered  when the cache size limit is reached above 4 for at least 5 minutes.
+  This alert is triggered  when the CPU Usage is exceeds 90% for at least 5 minutes.
```
Comment (mdumpati) [inline alert-policies/nextcloud/CPUUsage.yml:4]:
```suggestion
  This alert is triggered  when the CPU Usage exceeds 90% for at least 5 minutes.
```

File: alert-policies/nextcloud/MemoryUsage.yml [hunk 2]
```diff
@@ -1,11 +1,11 @@
-name: Heap Memory Usage
+name: Memory Usage 
 
 description: |+
-  This alert is triggered  when the Heap Memory Usage is below 2 for at least 5 minutes.
+  This alert is triggered  when the Memory Usage is exceeds 90% for at least 5 minutes.
```
Comment (mdumpati) [inline alert-policies/nextcloud/MemoryUsage.yml:4]:
```suggestion
  This alert is triggered  when the Memory Usage exceeds 90% for at least 5 minutes.
```

**Review by mdumpati — COMMENTED** [review]
@pkudikyala check the comments

File: alert-policies/nextcloud/CPUUsage.yml (hunk 1 above)
Comment (pkudikyala) [reply alert-policies/nextcloud/CPUUsage.yml]: Hi @mdumpati ,
As per your comments, I have made changes. Kindly review it.
Thanks...

File: alert-policies/nextcloud/MemoryUsage.yml (hunk 2 above)
Comment (pkudikyala) [reply alert-policies/nextcloud/MemoryUsage.yml]: Hi @mdumpati ,
As per your comments, I have made changes. Kindly review it.
Thanks...

**Review by rahulbasu-nr — APPROVED** [review]
Looks good to me.

**Pull request merged**